btree/btree.c

```diff
@@ -8445,199 +8445,6 @@ static void btree_ut_fini(void)
 	btree_ut_initialised = false;
 }
 
-#if 0
-static void get_rec_at_index(struct nd *node, int idx, uint64_t *key,
```

Comment:
`get_rec_at_index and get_key_at_index` are called from `ut_traversal()`.  Currently ut_traversal () is disabled but it is useful for debuging. Re-include these functions.

Reply:
I do not see anyone invoking ut_traversal. If this function is called from some other ut then we need to have had the call at least commented in the code. I will have a look at the traversal function and see if we need it. If we need it then we need to make a call to it from some function.